docs/components/sign-in.md

```diff
@@ -0,0 +1,221 @@
+# SignIn Component
+
+`SignIn` is a React higher order component that manages the user sign-in process, including handling status changes, user information updates, and error handling. It's designed to be flexible and customizable, allowing you to pass in custom components.
+
+## Props
+
+The `SignIn` component takes the following props:
+
+| Prop             | Type          | Default Value | Description |
+|------------------|---------------|---------------|-------------|
+| authStatus       | string        | 'Signed Out'  | The initial authentication status. |
+| connect          | object        | N/A           | The Greenruhm Connect SDK. In order for this component to work, it is necessary that the SDK is initialized and provided. |
+| ErrorComponent   | component     | N/A           | A component to display in case of an error. This prop will not be used in case `ViewComponent` is passed as well. |
+| FormComponent    | component     | N/A           | A component that renders the sign-up form. This prop will not be used in case `ViewComponent` is passed as well. |
+| LoadingComponent | component     | N/A           | A component to display while the sign-up process is in progress. This prop will not be used in case `ViewComponent` is passed as well. |
+| onSuccess        | function      | console.log() | A function that will be called upon successful sign-in. The default function logs the user data to the console. The `onSuccess` signature is as follows:<br>`const onSuccess = ({ authStatus, email, username }) => {}`.<br>Where:<br>- `authStatus` is `'Signed In'`<br>- `email` is the email the user provided<br> \- `username` is the username the user provided |
```

Comment:
I would link this to `onSuccess` function documentation below this table rather than trying to document `onSuccess` inline here.